system/jlib/jsocket.cpp

```diff
@@ -4415,7 +4514,8 @@ class CSocketEpollThread: public CSocketBaseThread
         validateerrcount = 0;
         offset = 0;
         selecttrace = trc;
-        epfd = ::epoll_create(XFD_SETSIZE);
+        max_ret_events = 5000;
```

Comment:
minor: perhaps should be a constant defined elsewhere (e.g. at head of file), with a comment describing significance.